### PR TITLE
Updates victoriametrics to be two separate packages for cluster vs standalone mode

### PR DESCRIPTION
**Pull request by dakaneye**

### CODE REVIEW

File: victoriametrics-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 115a76d28c3bb0e63685db1342f80df6af3033dd
+      repository: https://github.com/VictoriaMetrics/VictoriaMetrics
+      tag: v${{package.version}}-cluster
+
+subpackages:
+  - range: victoria-packages
+    name: ${{package.name}}-${{range.key}}
```
Comment (EyeCantCU) [inline victoriametrics-cluster.yaml:27]: Can we do `victoriametrics-${{range.key}}-cluster`?

File: victoriametrics-single.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package:
+  name: victoriametrics-single
```
Comment (EyeCantCU) [inline victoriametrics-single.yaml:2]: We could leave packages for single node configs just `victoriametrics` unless there are deviations in how these are named

File: victoriametrics-single.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        with:
+          tags: nethttpomithttp2
+          packages: ./app/${{range.key}}
+          output: ${{range.key}}
+          ldflags: -X github.com/VictoriaMetrics/VictoriaMetrics/lib/buildinfo.Version=${{range.key}}-$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")-$(git rev-parse HEAD)
+    test:
+      environment:
+        contents:
+          packages:
+            - busybox
```
Comment (EyeCantCU) [inline victoriametrics-single.yaml:38]: Busybox's inclusion is implicit. We don't need to add it

File: victoriametrics-single.yaml (hunk 2 above)
Comment (dakaneye) [reply victoriametrics-single.yaml:2]: There are two differing naming conventions in their repo and docker image tags:
* In the [repo:](https://github.com/VictoriaMetrics/VictoriaMetrics/tags) tags are v1.102.1 and v1.102.1-cluster
* In [dockerhub](https://hub.docker.com/r/victoriametrics/vmagent/tags?page=&page_size=&ordering=&name=): they use `cluster` and `public-single-node`

Comment (dakaneye) [reply victoriametrics-single.yaml]: removed the `single` suffix

File: victoriametrics.yaml [hunk 4]
```diff
@@ -6,6 +6,13 @@ package:
   copyright:
     - license: Apache-2.0
 
+data:
+  - name: victoria-packages
+    items:
+      victoria-metrics: VictoriaMetrics is a fast, cost-effective and scalable monitoring solution and time series database.
```
Comment (EyeCantCU) [inline victoriametrics.yaml:12]: I think we should retain this in the main package so that we don't end up with victoriametrics-victoria-metrics

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**